Applications/Tizen_CAPI/capi_func.c

```diff
@@ -299,75 +299,118 @@ int main(int argc, char *argv[]) {
 
   /* model create */
   status = ml_train_model_construct(&model);
-  NN_RETURN_STATUS();
+  if (status != ML_ERROR_NONE) {
+    return 1;
+  }
+
   /* input layer create */
   status = ml_train_layer_create(&layers[0], ML_TRAIN_LAYER_TYPE_INPUT);
-  NN_RETURN_STATUS();
+  if (status != ML_ERROR_NONE) {
+    NN_RETURN_STATUS();
+  }
 
   /* set property for input layer */
   status = ml_train_layer_set_property(layers[0], "input_shape=1:1:62720",
                                        "normalization=true",
                                        "bias_initializer=zeros", NULL);
-  NN_RETURN_STATUS();
+  if (status != ML_ERROR_NONE) {
+    ml_train_layer_destroy(layers[0]);
+    NN_RETURN_STATUS();
+  }
 
   /* add input layer into model */
   status = ml_train_model_add_layer(model, layers[0]);
-  NN_RETURN_STATUS();
+  if (status != ML_ERROR_NONE) {
+    ml_train_layer_destroy(layers[0]);
+    NN_RETURN_STATUS();
+  }
 
   /* create fully connected layer */
   status = ml_train_layer_create(&layers[1], ML_TRAIN_LAYER_TYPE_FC);
-  NN_RETURN_STATUS();
+  if (status != ML_ERROR_NONE) {
+    ml_train_layer_destroy(layers[1]);
```

Comment:
destroy layer[0] as well.

Reply:
It is already added to the model so do not need to destroy.

Reply:
`    ml_train_layer_destroy(layers[1]);` gone

Reply:
yes